gitalias.txt

```diff
@@ -13,6 +13,7 @@
   d = diff
   f = fetch
   g = grep
+  h = push
```

Comment:
For me `git h` looks too much like `git help`.

Reply:
That's a good point. Do you think there would be a better letter for `push`?

Reply:
Letter `u` perhaps. The namespace is pretty limited. :-)

Reply:
Yes it is :) I was thinking `u` but then I always got that confused with `pull`.